vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -0,0 +1,142 @@
+"""Unit tests for hyphen.models.slider."""
+import json
+
+import plotly
+import pytest
+from dash import dcc
+from pydantic import ValidationError
+
+import vizro.models as vm
+from vizro.actions import export_data
+from vizro.models._action._actions_chain import ActionsChain
+
+
+class TestSliderInstantiation:
+    """Tests model instantiation."""
+
+    def test_create_default_slider(self):
+        slider = vm.Slider()
+
+        assert hasattr(slider, "id")
+        assert slider.type == "slider"
+        assert slider.min is None
+        assert slider.max is None
+        assert slider.step is None
+        assert slider.marks is None
+        assert slider.value is None
+        assert slider.title is None
+        assert slider.actions == []
+
+    @pytest.mark.parametrize(
+        "min, max, step, marks, value, title",
+        [
+            (0, 10, 1, {}, 3, "Test text"),
+            (100, 200, 50, {}, 100, """## Test header"""),
+            (1.23, 6.78, 0.01, {}, None, 1.23),
+            (0, 10, None, {str(i): i for i in range(0, 10, 1)}, 5, ""),  # add marks
+        ],
+    )
+    def test_create_slider_with_optional(  # noqa
+        self, min, max, step, marks, value, title
+    ):
+        slider = vm.Slider(
+            min=min, max=max, step=step, marks=marks, value=value, title=title
+        )
+
+        assert slider.min == min
+        assert slider.max == max
+        assert slider.step == step
+        assert slider.marks == marks
+        assert slider.value == value
+        assert slider.title == str(title)
+
+    def test_invalid_reverse_slider(self):
```

Comment:
Maybe change this to `test_create_slider_invalid_min_max` as to be more verbose.